apps/src/signUpFlow/LoginTypeSelection.tsx

```diff
@@ -60,6 +61,24 @@ const LoginTypeSelection: React.FunctionComponent = () => {
     }
   }, [passwordIcon, confirmPasswordIcon, emailIcon]);
 
+  useEffect(() => {
+    const handleKeyDown = (event: KeyboardEvent) => {
+      if (event.key === 'Enter') {
+        const button = screen.getByRole('button', {
```

Comment:
It seems like the preferred method is to use a ref= attribute on the button, but our design system component doesn't currently support that